<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="Pomodoro">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>581</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="pagePomodoro">
     <widget class="QPushButton" name="btnStartPomodoro">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>START!</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTimer">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>271</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic</family>
        <pointsize>72</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>TIME</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressPomodoro">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>212</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPausePomodoro">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PAUSE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnResetPomodoro">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>270</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>RESET</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnConfig">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>310</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Configurations</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelStatus">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageConfig">
     <widget class="QPushButton" name="btnGoBack">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>310</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>READY!</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>161</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="TimeShortBreak">
       <item row="1" column="0">
        <widget class="QTimeEdit" name="timeSB"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="timeSBL">
         <property name="text">
          <string>Short Break Time</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>161</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="PomodoroTime">
       <item row="1" column="0">
        <widget class="QTimeEdit" name="timePom"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="timePomL">
         <property name="text">
          <string>Pomodoro Time</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>161</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="TimeLongBreak">
       <item row="1" column="0">
        <widget class="QTimeEdit" name="timeLB"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="timeLBL">
         <property name="text">
          <string>Long Break Time</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>583</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPomodoroProject">
    <property name="title">
     <string>PomodoroProject</string>
    </property>
   </widget>
   <addaction name="menuPomodoroProject"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
